docs/_docs/0.5.0/2_4_configurations.md

```diff
@@ -391,29 +391,65 @@ Property: `hoodie.compaction.payload.class` <br/>
 
 
 ### Metrics configs
```

Comment:
if you dohn't mind, lets add a link to this section from this page.. http://hudi.apache.org/docs/deployment.html#monitoring ?

Reply:
> if you dohn't mind, lets add a link to this section from this page.. http://hudi.apache.org/docs/deployment.html#monitoring ?

well, let me change it.

Reply:
did it update?